### PR TITLE
adds simple version of systray available in debug mode and for kolide employees

### PR DESCRIPTION
**Pull request by James-Pickett**
* adds systray icon that runs as separate process
* systray exits when parent process exits
* only runs when in debug mode or for kolide employees

image

### CODE REVIEW

File: cmd/launcher/systray/systray.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package systray
+
+import (
+	_ "embed"
```
Comment (directionless) [inline cmd/launcher/systray/systray.go:4]: Is this used?

**Review by directionless — APPROVED** [review]
There's a lot here I'm unsure of. But I think this is the right starting point.

File: cmd/launcher/systray/systray.go (hunk 1 above)
Comment (James-Pickett) [reply cmd/launcher/systray/systray.go:4]: yes it has to be imported in order to use the `go:embed` directive in
```
//go:embed kolide-mark-only-white.png
var kolideSystrayIcon []byte
```
but if you name the import you'll get an error that it's not in use

Comment (directionless) [reply cmd/launcher/systray/systray.go:4]: That's not in this file? (I think it moved to icon files?)

Comment (James-Pickett) [reply cmd/launcher/systray/systray.go:4]: ohh woops, that's me not paying attention to the file the comment was in =), I'll remove it

File: cmd/launcher/systray/icon_unix.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+//go:build darwin || linux
+// +build darwin linux
```
[the remark this answers is not in the document]
Comment (James-Pickett) [reply cmd/launcher/systray/icon_unix.go]: changed to icon.go

**Review by directionless — APPROVED** [review]

**Comment by James-Pickett** [comment]
closing this while I run CI in my forked repo

**Pull request closed without merging**